docs/ml-features.md

```diff
@@ -183,6 +183,78 @@ for words_label in wordsDataFrame.select("words", "label").take(3):
 </div>
 </div>
 
+## PolynomialExpansion
+
+[Polynomial expansion](http://en.wikipedia.org/wiki/Polynomial_expansion) is the process of expanding your features into a polynomial space, which is formulated by an n-degree combination of original dimensions. A [PolynomialExpansion](api/scala/index.html#org.apache.spark.ml.feature.PolynomialExpansion) class provides this functionality.  The example below shows how to expand your features into a 3-degree polynomial space.
+
+Note: We do not provide Python API for `PolynomialExpansion` in this version.
+
+<div class="codetabs">
+<div data-lang="scala" markdown="1">
+{% highlight scala %}
+import org.apache.spark.ml.feature.PolynomialExpansion
+
+val data = Array(
+  Vectors.sparse(3, Seq((0, -2.0), (1, 2.3))),
+  Vectors.dense(-2.0, 2.3),
```

Comment:
Length should match other data.  Also, can the data be made as simple as possible, such as 3 length-2 vectors?  (same for Java example)

Alternatively, you could load data via MLUtils.loadLibSVMFile